backend/mora/service/configuration_options.py

```diff
@@ -0,0 +1,184 @@
+#
+# Copyright (c) Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import psycopg2
+import flask
+from mora import exceptions
+from .. import settings
+from .. import util
+
+
+blueprint = flask.Blueprint('configuration', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+def _get_connection():
+    conn = psycopg2.connect(user=settings.USER_SETTINGS_DB_USER,
+                            dbname=settings.USER_SETTINGS_DB_NAME,
+                            host=settings.USER_SETTINGS_DB_HOST,
+                            port=settings.USER_SETTINGS_DB_PORT,
+                            password=settings.USER_SETTINGS_DB_PASSWORD)
+    return conn
+
+
+@blueprint.route('/ou/<uuid:unitid>/configuration', methods=['POST'])
+@util.restrictargs()
+def set_org_unit_configuration(unitid):
+    """Set a configuration setting for an ou.
+
+    .. :quickref: Unit; Create configuration setting.
+
+    :statuscode 201: Setting created.
+    :statuscode 404: No such unit found.
+
+    :param unitid: The UUID of the organisational unit to be configured.
+
+    :<json object conf: Configuration option
+
+    .. sourcecode:: json
+
+      {
+        "org_units": {
+          "show_location": "True"
+        }
+      }
+
+    :returns: True
+    """
+    conn = _get_connection()
+    try:
+        cur = conn.cursor()
+
+        configuration = flask.request.get_json()
+        orgunit_conf = configuration['org_units']
+
+        for key, value in orgunit_conf.items():
+            query = ("SELECT id FROM orgunit_settings WHERE setting = %s " +
+                     "AND object=%s")
+            cur.execute(query, (key, unitid))
+            rows = cur.fetchall()
+            if not rows:
+                query = ("INSERT INTO orgunit_settings (object, setting, " +
+                         "value) VALUES (%s, %s, %s)")
+                cur.execute(query, (unitid, key, value))
+            elif len(rows) == 1:
+                query = "UPDATE orgunit_settings SET value=%s WHERE id=%s"
+                cur.execute(query, (value, rows[0][0]))
+            else:
+                exceptions.ErrorCodes.E_INCONSISTENT_SETTINGS(
+                    'Inconsistent settings for {}'.format(unitid)
+                )
+            conn.commit()
+    finally:
+        conn.close()
+    return flask.jsonify(True)
+
+
+@blueprint.route('/ou/<uuid:unitid>/configuration', methods=['GET'])
+@util.restrictargs()
+def get_org_unit_configuration(unitid):
```

Comment:
Split the body of this function out, in a similar fashion to the `set` functions.